src/Autofac.Integration.WebApi/Autofac.Integration.WebApi.csproj

```diff
@@ -1,138 +1,65 @@
-    <Compile Include="AutofacWebApiModelBinderProvider.cs" />
-    <Compile Include="ControllerTypeKey.cs" />
-    <Compile Include="AuthenticationFilterWrapper.cs" />
-    <Compile Include="AuthenticationFilterOverrideWrapper.cs" />
-    <Compile Include="AuthorizationFilterOverrideWrapper.cs" />
-    <Compile Include="ExceptionFilterOverrideWrapper.cs" />
-    <Compile Include="FilterPredicateMetadata.cs" />
-    <Compile Include="HttpConfigurationExtensions.cs" />
-    <Compile Include="HttpRequestMessageProvider.cs" />
-    <Compile Include="IAutofacContinuationActionFilter.cs" />
-    <Compile Include="IAutofacAuthenticationFilter.cs" />
-    <Compile Include="CurrentRequestHandler.cs" />
-    <Compile Include="DependencyResolverExtensions.cs" />
-    <Compile Include="ExceptionFilterWrapper.cs" />
-    <Compile Include="IAutofacActionFilter.cs" />
-    <Compile Include="IAutofacAuthorizationFilter.cs" />
-    <Compile Include="IAutofacExceptionFilter.cs" />
-    <Compile Include="FilterMetadata.cs" />
-    <Compile Include="AutofacControllerConfigurationAttribute.cs" />
-    <Compile Include="AutofacControllerConfigurationAttributeResources.Designer.cs">
+    
+  <ItemGroup>    
+    <Compile Update="AutofacControllerConfigurationAttributeResources.Designer.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>AutofacControllerConfigurationAttributeResources.resx</DependentUpon>
     </Compile>
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="RegistrationExtensions.cs" />
-    <Compile Include="RegistrationExtensionsResources.Designer.cs">
+    <Compile Update="RegistrationExtensionsResources.Designer.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>RegistrationExtensionsResources.resx</DependentUpon>
     </Compile>
   </ItemGroup>
+
   <ItemGroup>
-    <EmbeddedResource Include="AutofacControllerConfigurationAttributeResources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>AutofacControllerConfigurationAttributeResources.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
-    <EmbeddedResource Include="RegistrationExtensionsResources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>RegistrationExtensionsResources.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="Autofac.Integration.WebApi.nuspec" />
+    <None Include="..\..\build\icon.png" Pack="true" PackagePath="\" />
   </ItemGroup>
+  
   <ItemGroup>
-    <PackageReference Include="Autofac">
-      <Version>5.2.0</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.AspNet.WebApi.Core">
-      <Version>5.2.0</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers">
-      <Version>2.6.1</Version>
+    <PackageReference Include="Autofac" Version="[5.0.0, 7.0.0)" />
```

Comment:
[In Autofac.Mvc we ended up going with straight 6.0+.](https://github.com/autofac/Autofac.Mvc/blob/70c77d1485b00797332560fa31e7a435ed4a63f6/src/Autofac.Integration.Mvc/Autofac.Integration.Mvc.csproj#L34) Should we do that here, too?

Reply:
Yes, we should be consistent, especially among the tightly couple Mvc+WebApi projects. Will fix.